compiler/rustc_index/src/lib.rs

```diff
@@ -1,9 +1,9 @@
 // tidy-alphabetical-start
+#![cfg_attr(all(feature = "nightly", bootstrap), feature(new_zeroed_alloc))]
```

Comment:
This doesn't seem right to me. I'd expect just cfg(bootstrap) here -- presumably, once this is released/used via nightly feature gate this PR would have already landed and so the feature isn't needed.